lib/subvalid/validators/length_validator.rb

```diff
@@ -4,11 +4,19 @@ class LengthValidator
       def self.validate(object, validation_result=ValidationResult.new, *args)
         return unless object
         args = args.to_h
+        message = args.delete(:message)
```

Comment:
does the new option need to be added to any docs?

Reply:
Ah yes, I'll update the Readme.

Reply:
Resolved in db7ddb8